solidity/contracts/NetworkSettings.sol

```diff
@@ -30,23 +30,41 @@ contract NetworkSettings is INetworkSettings, Owned, Utils {
         return (_networkFeeWallet, _networkFee);
     }
 
+    /**
+     * @dev returns the network fee wallet
+     *
+     * @return network fee wallet
+     */
+    function networkFeeWallet() external view override returns (address) {
```

Comment:
Perhaps `feeParams` should be renamed `networkFeeParams` so that it's consistent with the new getters?

Reply:
makes sense

Reply:
In that case, we may as well name the contract `NetworkFee`.

Reply:
it's gonna hold other global network settings, so don't think so. but feeParams should be renamed.